### ISSUE TITLE
Autoformatting breaks Dockerfile with heredoc (for COPY instruction)

### ISSUE THREAD

**Issue by moha-gh** [microsoft/vscode-docker#3950]
Hi,

I have a problem that is very similar to #3327. Using `v1.25.1` of the extension, a simple `Dockerfile` that uses [heredocs](https://www.docker.com/blog/introduction-to-heredocs-in-dockerfiles/) in `COPY` instructions like this:

```
FROM alpine:3.18

COPY <<EOF /foo/bar
x
y
z
EOF

COPY <<EOF /bar/foo
a
b
c
EOF

```

is re-formatted to

```
FROM alpine:3.18

COPY <<EOF /foo/bar
    x
    y
    z
    EOF

COPY <<EOF /bar/foo
    a
    b
    c
    EOF

```

which breaks things completely (due to the indentation of the `EOF` marker).

From the previously mentioned bug report I gather that https://github.com/rcjsuen/dockerfile-utils is used for formatting. If I use the latest upstream release ([0.11.0](https://github.com/rcjsuen/dockerfile-utils/releases/tag/v0.11.0))  via `docker run rcjsuen/dockerfile-utils`, the file is formatted correctly.

Would it be possible to pull in the newer version? Between [0.9.0](https://github.com/microsoft/vscode-docker/blob/c77c14fa57d6cd25f6543046fe2d58440a029c02/package-lock.json#L2381) which is in use right now AFAIU and 0.11.0 there have been a number of heredoc-related fixes (see [here](https://github.com/rcjsuen/dockerfile-utils/compare/v0.9.0...v0.11.0)) - and other as well.

**Comment by bwateratmsft** [comment]
Unfortunately there isn't a version of `dockerfile-language-server-nodejs` / `dockerfile-language-service` that uses v0.11.0 of the `dockerfile-utils` package. So no compatible version exists. It's possible we could override those two packages to force them to use `dockerfile-utils` 0.11.0, assuming it actually does work.

**Comment by alexyaang** [comment]
The fix has now been released in Docker extension version 1.26.0
